cpp/src/io/csv/csv_gpu.cu

```diff
@@ -580,10 +957,8 @@ cudaError_t __host__ DetectColumnTypes(const char *data,
                                        cudaStream_t stream)
 {
   // Calculate actual block count to use based on records count
-  int blockSize   = 0;  // suggested thread count to use
-  int minGridSize = 0;  // minimum block count required
-  CUDA_TRY(cudaOccupancyMaxPotentialBlockSize(&minGridSize, &blockSize, dataTypeDetection));
-  const int gridSize = (num_rows + blockSize - 1) / blockSize;
+  const int blockSize = csvparse_block_dim;
+  const int gridSize  = (num_rows + blockSize - 1) / blockSize;
```

Comment:
```suggestion
  const cudf::experimental::detail::grid_1d grid{num_rows, block_size, 1};
```

Reply:
I prefer the original code here

Reply:
I know that, and in most of my life I embrace diversity. But in libcudf I embrace consistency.

Reply:
You mean you demand conformity ?